site_root/input/resources/examples/Observation-se-001.json

```diff
@@ -0,0 +1,76 @@
+{
+  "resourceType": "Observation",
```

Comment:
Could you put the followings below `id`:

```json
"meta": {
  "profile": [
    "http://hl7.org/fhir/StructureDefinition/Observation"
  ]
}
```

This must be why the issue pointed out above happened.